docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -74,18 +75,61 @@
 
 ## Python
 
-This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Ethereum network.
+This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Kovan testnet.
 
 ```python Kovan
 {% include 'samples/PriceFeeds/PriceConsumerV3.py' %}
 ```
 
+{% endmarkdown %}
+
 <div class="row cl-button-container">
     <div class="col-xs-12 col-md-12">
     <a href="https://repl.it/@alexroan/GetLatestPriceWeb3PY" class="cl-button--ghost python-tracked">Run this Python example</a>
     </div>
 </div>
 
+{% markdown %}
+
+## Golang
+
+This example uses [go-ethereum](https://github.com/ethereum/go-ethereum) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Kovan testnet.
+In order to interact with a smart contract in go, you need to:
+
+1. Generate the [Contract ABI](https://docs.soliditylang.org/en/latest/abi-spec.html) from a solidity source file using [solc](https://docs.soliditylang.org/en/latest/using-the-compiler.html). In this case, we are going to download the [`AggregatorV3Interface`](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol) and generate an ABI called `AggregatorV3Interface.abi`:
+
+```shell 
+wget https://raw.githubusercontent.com/smartcontractkit/chainlink/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol
+solc --abi AggregatorV3Interface.sol -o .
+```
+
+2. Convert `AggregatorV3Interface.abi` to a `Go` file `aggregator_v3_interface.go`. Ensure that [Geth](https://geth.ethereum.org/docs/install-and-build/installing-geth) is installed so you can use `abigen`. This source code generator allows to convert Ethereum contract definitions into easy to use, compile-time type-safe Go packages.
```

Comment:
Recommend disambiguating the language here by using terminology consistent with Geth abigen util. For example " generate the Go bindings required to programmatically interact with a smart contract using generated, typesafe Go code. This is done using the abigen util" and link abigen to: https://geth.ethereum.org/docs/dapp/native-bindings

Reply:
ok thanks. will be resolved in the next push